galley/pkg/config/analysis/analyzers/service/portname.go

```diff
@@ -72,6 +72,11 @@ func (s *PortNameAnalyzer) analyzeService(r *resource.Instance, c analysis.Conte
 			m := msg.NewPortNameIsNotUnderNamingConvention(
 				r, port.Name, int(port.Port), port.TargetPort.String())
 
+			if svc.Type == "ExternalName" {
```

Comment:
Should we skip warning on `NewPortNameIsNotUnderNamingConvention` if svc type is external name? Otherwise we are generating two somewhat duplicated warnings for an ExternalName svc.

Reply:
It is an error message should I convert it to a warning only?. 

Actually, NewPortNameIsNotUnderNamingConvention is skipped is service type is ExternalName and port does not follow the naming convention. No duplicate warnings are displayed

Reply:
Tested this and no duplicated message shows. The message gets assigned issue ExternalNameServiceTypeInvalidPortName when svc is of type external name